### ISSUE TITLE
Bugs in Advanced options in cooperation with screen readers

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#1124]
**Reported by krzyszp on 18 Dec 41858584 22:50 UTC**
In "Advanced/Select computer" window labels "Host name" and "Password" are not readable by screen readers

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1151

**Comment by romw** [comment]
**Commented by romw on 16 Apr 41858682 03:28 UTC**
I'm afraid I'm not able to reproduce this issue.

According to the !AccCheck tool:  http://acccheck.codeplex.com/

Both the host name and password controls are exposed to both the MSAA interface and the newer MSUIA interface.

**Comment by ChristianBeer** [comment]
Should be checked against current 7.6.33

also related #1088 #1123

**Comment by Ageless93** [comment]
Can someone check this with BOINC version 7.6.33 or newer, please?

**Comment by Ageless93** [comment]
Can someone with BOINC 7.8.2 or 7.8.3 check this? 
Or else point out a screen reader so we can test this ourselves?
